### DISCUSSION TITLE
ngrinder 실행 시, svn 연동이 자동으로 되는걸까요?

### DISCUSSION THREAD

**Discussion by perfTestHY** [naver/ngrinder#902] [Q&A] [answered]
리눅스 서버에 svn을 yum 으로 설치하고, 3690 포트 활성화 된거 확인한 상태에서

ngrinder를 톰캣으로 띄우고, 스크립트 작성 쪽에 있는 svn 주소로 연동하려고 하니 연결이 되고 있지 않습니다.

기존에 다른 테스트 진행할때는 이런식으로 해서 정상적으로 intellij로 gradle연동했던걸로 기억하는데, 혹시 svn 연동하는 방법이 따로 있을까요?

**Comment by donggyu04** [comment, marked as answer]
ngrinder 는 외부 svn 서버와의 연동을 지원하지 않습니다.
ngrinder 컨트롤러 구동시 컨트롤러 자체가 svn 서버로써 동작하게 됩니다. 따라서 별도 svn 서버 구성 없이 사용가능합니다.
